db/migrate/20220616112301_change_nullable_lighthouse_sample_mongodb_id.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+#
+# This migration disables NOT NULL to the mongodb_id column because the plan is
+# this table will be maintained by GSU team and they won't use that column anymore
+class ChangeNullableLighthouseSampleMongodbId < ActiveRecord::Migration[6.0]
+  def change
+    change_column_null(:lighthouse_sample, :mongodb_id, true)
+  end
```

Comment:
Have you tested whether reversing the migration works for this? (Just in case need to roll back)

Reply:
good point, no, but I'll do it now

Reply:
oh no! I've just realised that the database has mongodb_id as nullable. So this pull request is not needed :(